lib/widgets/verse_card.dart

```diff
@@ -90,6 +90,7 @@ class VerseCard extends StatelessWidget {
                                             where: 'reference = ?',
                                             whereArgs: [_reference]),
                                       );
+                                      //TODO: Snackbar favorited message?
```

Comment:
Would you mind changing this to `// TODO(Robotboy897): Snackbar favorited message?` so that way I know you left it? (I'll do that for my todos as well in a future commit, so we know who left each one.)

Reply:
Yup, done!